src/app/services/dovetail-api.service.ts

```diff
@@ -5,41 +5,33 @@ import {Observable} from 'rxjs/Observable';
 
 import {AdzerkNativeAdAPIRequest} from '../services/adzerk_native_ad_api_client';
 
+export interface DovetailArrangementEntry {
+  id: string;
+  type: string;
+}
+
+export interface DovetailDebugResponse {
+  arrangement: DovetailArrangementEntry[];
+  program: {id: string};
+  request: AdzerkNativeAdAPIRequest;
+  warning: string;
+  error: string;
+}
+
 @Injectable()
 export class DovetailService {
   constructor (private http: Http) {}
 
-  getAdzerkRequestBody(url: string): Observable<AdzerkNativeAdAPIRequest> {
```

Comment:
oops, spoke too soon in approval. This method is still being used here: https://github.com/PRX/backsaw.prx.org/blob/master/src/app/advanced/advanced.component.ts#L34